src/uhi/typing/plottable.py

```diff
@@ -86,6 +86,11 @@ def __eq__(self, other: Any) -> bool:
         Required to be sequence-like.
         """
 
+    def __iter__(self) -> Iterator[T]:
+        """
+        Useful element of a Sequence to include.
+        """
```

Comment:
This is the addition in answer to #6.

Reply:
@HDembinski, you've already okay'd the np.ndarray return, how about this? Are you okay with requiring any `axes` also be iterable? Once 3.6 is dropped we could further restrict to axes being a Sequence, or we could manually add all the parts of the Sequence protocol (there are only a handful more), but this is probably the most important part of a sequence and will simplify downstream code that currently must use `__len__` and `__getitem__`. I believe @jpivarski has more-or-less okay'd both. (though please speak up if not!)

Reply:
(note: the container `.axes` holding Axis objects is what's being discussed here, not the Axis objects themselves.)

Reply:
Sure, this makes sense. Iterable is safe. I am not sure whether we want to require full Sequence support. `__reversed__` and `count` does not make much sense.

Reply:
> (note: the container .axes holding Axis objects is what's being discussed here, not the Axis objects themselves.)

Ahh, I made a mistake. This _is_ the Axis object. We already require the axes to be in a Sequence (which is why we require `__eq__` on an Axis).  I think this is still fine, but requires more thought that I first expected. I think that something that provides `__getitem__` and `__len__` should likely provide an `__iter__`; we do in boost-histogram (and it's trivial to provide if you have the other two). I caught it with the NumPyPlottibleHistogram, which is a minimal, unfillable histogram that adapts other histogram types to the PlottableHistogram protocol. It didn't have an `__iter__`, though I'm happy to add it. Any second thoughts, @HDembinski or @jpivarski ?

Reply:
Sounds good.

Reply:
Thanks, sorry for the confusion!